src/main/java/com/puppycrawl/tools/checkstyle/utils/CheckUtil.java

```diff
@@ -252,6 +250,18 @@ public static DetailAST getFirstNode(final DetailAST node) {
         return currentNode;
     }
 
+    /**
+     * Retrieves whether the current node is located before the next node.
+     * @param currentNode the current node.
+     * @param nextNode the next node to check.
+     * @return true, if the current node is located before the next node.
+     */
+    public static boolean isBeforeInSource(DetailAST currentNode, DetailAST nextNode) {
```

Comment:
Let's avoid in util method any names like current and next. Let's name them ast1 and ast2.